### DISCUSSION TITLE
How do I run a single task?

### DISCUSSION THREAD

**Discussion by SandraGH5** [flyteorg/flyte#1060] [Q&A] [answered]

**Comment by SandraGH5** [comment, marked as answer]
For information on running single tasks, please refer to https://docs.flyte.org/en/latest/howto/execute_single_task.html

**Reply by ncouronne** [reply]
Hi,
This page doesn't exist anymore

**Reply by samhita-alla** [reply]
@ncouronne You can refer to https://github.com/flyteorg/flytesnacks/pull/330. It should be merged soon!
